runtime/vm/stringhelpers.cpp

```diff
@@ -445,14 +434,10 @@ verifyQualifiedName(J9VMThread *vmThread, j9object_t string, UDATA allowedBitsFo
 	}
 
 	/* map the return code against the arity value from checkNameImpl (called by bcvCheckClassName):
-	 * 1) if arity = -1, then rc = CLASSNAME_INVALID(0)
-	 * 2) if arity = 0,  then rc = CLASSNAME_VALID_NON_ARRARY(1)
-	 * 3) if arity > 0,  then rc = CLASSNAME_VALID_ARRARY(2)
+	 * 1) if arity = 0,  then rc = CLASSNAME_VALID_NON_ARRARY(1)
```

Comment:
Please don't include the constant values in the comments.

Reply:
Remove these constant values and modified the comments.